appConfig/appConfig.snd4.yaml

```diff
@@ -1,3 +1,9 @@
 - key: APPINSIGHTS_CONNECTIONSTRING
   value: "{{servicename}}-APPINSIGHTS-CONNECTIONSTRING"
-  type: "keyvault"
\ No newline at end of file
+  type: "keyvault"
+- key: MESSAGE_HOST
+  value: "SNDADPINFSB4401.servicebus.windows.net"
+- key: CRM_TOPIC_ADDRESS
+  value: "fcp-sfd-crm-events"
```

Comment:
Hoe about fcp-sfd-crm-messages?

Reply:
Agree, was thinking something specifying 'messages' would make the most sense since our CRM service will be linked to our messages processor (`ffc-sfd-messages-processor`). I have made [the updates to the topic name]() in the `appConfig` for SND4. I will also update our `ffc-azure-service-bus-scripts` repo so that local topic/subscription matches the deployed values.